tools/ovc/openvino/tools/ovc/moc_frontend/pytorch_frontend_utils.py

```diff
@@ -90,76 +91,84 @@ def get_value_from_list_or_dict(container, name, idx):
     return None
 
 
+def flatten_inputs(inputs, names=None):
+    flattened = []
+    if isinstance(inputs, dict):
```

Comment:
should we somehow take care in case if we have several dicts with the same keys?

Reply:
I am still not sure how we can resolve that. Do we really have such issue in real models?